.github/workflows/conda-package.yml

```diff
@@ -8,50 +8,53 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
+        include:
+        - os: ubuntu-latest
+          pkgs_dirs: ~/.conda/pkgs
+          condarc: ~/.condarc
+          conda-bld: /usr/share/miniconda/conda-bld/linux-64/
+        - os: windows-latest
+          pkgs_dirs: C:\Users\runneradmin\.conda\pkgs
```

Comment:
Can we use Windows env variables like `%HOMEDRIVE%%HOMEPATH%` to replace hard-coded home path which may change down the road?

Reply:
There is currently no way to use environment variables here